src/components/common/Cart/Cart.tsx

```diff
@@ -0,0 +1,20 @@
+import { FC } from "react"
+import css from "./cart.module.css"
+
+interface CartProps {
+       image: string;
+       title: string;
+       description: string;
+       price: string;
+}
+
+export const Cart: FC<CartProps> = ({ image, title, price, description }) => {
```

Comment:
карточка это Card, Cart - это корзина
```suggestion
export const Card: FC<CardProps> = ({ image, title, price, description }) => {
```